### ISSUE TITLE
dc:date vs oio:date_created for new terms

### ISSUE THREAD

**Issue by matentzn** [information-artifact-ontology/ontology-metadata#63]
moved from https://github.com/OBOFoundry/OBOFoundry.github.io/issues/906

**Comment by cmungall** [comment]
also need to harmonize on range: individual or URI Literal or CURIE string (GO) or full-name-with-spaces literals (OBI)

**Comment by zhengj2007** [comment]
See https://github.com/information-artifact-ontology/ontology-metadata/issues/52

If we decide to use dc:date, is it the date that the term created or last updated? Do we need two date APs, one for creation date and one for last updated date.

**Comment by matentzn** [comment]
I dont think we should derive an answer to @zhengj2007 question from current practice, but I would like to suggest to use dcterms:date for last updated and dcterms:created for creation date, see https://www.dublincore.org/specifications/dublin-core/dcmi-terms/
